.github/workflows/tests_linux.yml

```diff
@@ -118,13 +119,14 @@ jobs:
           pytest tests/ $COVERAGE_FLAGS
           pl-device-test --device lightning.qubit --skip-ops --shots=20000 $COVERAGE_FLAGS --cov-append
           pl-device-test --device lightning.qubit --shots=None --skip-ops $COVERAGE_FLAGS --cov-append
+          mv coverage.xml coverage-${{ github.job }}.xml
 
-      - name: Upload coverage to Codecov
-        uses: codecov/codecov-action@v3
+      - name: Upload code coverage results
+        uses: actions/upload-artifact@v3
+        if: always()
```

Comment:
Do we need to specify always? Or is this to make it easy to temporarily disable in a given PR?

Reply:
I just copied it from somewhere... It seems like that `upload-artifact@v3` doesn't even explain what `always()` does. I will remove this.

Reply:
Ah, this is not for `upload-artifact` but for this action. Then it is not needed for this. I just copied it from the script uploading test result, which must be uploaded when a test failed.